src/librustc_typeck/check/method/suggest.rs

```diff
@@ -241,7 +241,8 @@ impl<'a, 'gcx, 'tcx> FnCtxt<'a, 'gcx, 'tcx> {
 
             MethodError::Ambiguity(sources) => {
                 let mut err = struct_span_err!(self.sess(), span, E0034,
-                                               "multiple applicable items in scope");
+                                 "multiple applicable items in scope");
```

Comment:
You can commit then squash. It'll be faster. If you don't know how to squash, take a look [here](https://blog.guillaume-gomez.fr/Git-tips/1/2).